lms/templates/dashboard.html

```diff
@@ -167,20 +163,23 @@
                     if not next_session:
                         continue
                     enrollment = CourseEnrollment(user=user, course_id=next_session['key'], mode=next_session['type'])
+                  # Only show email settings for entitlement cards if the entitlement has an associated enrollment
+                  show_email_settings = entitlement_session and (entitlement_session in show_email_settings_for)
 
                 session_id = enrollment.course_id
                 show_courseware_link = (session_id in show_courseware_links_for)
                 cert_status = cert_statuses.get(session_id)
                 can_refund_entitlement = entitlement and entitlement.is_entitlement_refundable()
                 can_unenroll = (not cert_status) or cert_status.get('can_unenroll') if not unfulfilled_entitlement else False
                 credit_status = credit_statuses.get(session_id)
-                show_email_settings = (session_id in show_email_settings_for)
+                if not entitlement:
+                  show_email_settings = (session_id in show_email_settings_for)
                 course_mode_info = all_course_modes.get(session_id)
                 is_paid_course = True if entitlement else (session_id in enrolled_courses_either_paid)
                 is_course_blocked = (session_id in block_courses)
                 course_verification_status = verification_status_by_course.get(session_id, {})
                 course_requirements = courses_requirements_not_met.get(session_id)
-                related_programs = inverted_programs.get(unicode(entitlement.course_uuid if is_unfulfilled_entitlement else session_id))
+                related_programs = inverted_programs.get(unicode(entitlement.course_uuid if entitlement else session_id))
```

Comment:
Why did this change?

Reply:
I'm not sure, I also don't believe I changed it. It should be was it was before though. I'll change it back.